templates/zerver/register.html

```diff
@@ -23,61 +23,108 @@ <h1>You&rsquo;re almost there.</h1>
+                        </div>
+                        <div id="organization_type_select_section" class="input-group inline-block">
+                            <select class="organization_type" name="organization_type" id="organization_type_select">
+                                <option value="Corporate">{{_('Corporate')}}</option>
+                                <option value="Open-source">{{_('Open Source')}}</option>
+                                <option value="Education">{{_('Education')}}</option>
+                                <option value="Non-profit">{{_('Non-Profit')}}</option>
+                                <option value="Other">{{_('Other')}}</option>
+                            </select>
+                        </div>
                     </div>
-                </div>
+                    <div id="offer_details">
+                        <div class="input-box">
+                            <div class="help-box margin-top" id="offer_details_help_box"></div>
+                            <div id="open_source_offer">
+                                <div class="input-box">
+                                    <label for="repository_link_label" class="inline-block label-title">{{ _('Repository Link') }}</label>
+                                    <div class="inline-block relative">
+                                        <input id="repository_link_label" class="required" type="text"
+                                          name="repository_link" required />
+                                    </div>
+                                </div>
+                            </div>
+                            <div id="non_profit_offer">
+                                <div class="input-box">
+                                    <label for="organization_nature_label" class="inline-block label-title">{{ _('Briefly describe the nature of your organization') }}</label>
+                                    <div class="inline-block relative">
+                                        <input id="organization_nature_label" class="required" type="text"
+                                          name="organization_nature" required />
+                                    </div>
+                                </div>
+                                <div class="input-box">
+                                    <label for="organization_website_label" class="inline-block label-title">{{ _('Website of your organization') }}</label>
+                                    <div class="inline-block relative">
+                                        <input id="organization_website_label" class="required" type="text"
+                                          name="organization_website" required />
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    {% endif %}
```

Comment:
I think it might be more readable to put this new set of options content in another template that we include into `register.html`; there's a lot of indentation here, and this bundle of options is a bit different than the rest.